src/types/react-interface.d.ts

```diff
@@ -13,7 +13,8 @@ type BreakPoint = {
 /* Typedef statements in C0 source code */
 type TypeDefInfo = {
     source: string, /* source: the original type being aliased */
-    key: number     /* key: the key of editor where this typedef info entry comes from */
+    alias: string,  /* alias: the alias type name in source code */
+    // key: number     /* key: the key of editor where this typedef info entry comes from */

```

Comment:
The `key` in `TypeDefInfo` is removed as now we are extracting `typedef` information only when the code is compiled and there's no need to do "incremental update" anymore.